content.js

```diff
@@ -15,7 +15,7 @@ stats.getDomainDistance(websiteDomain).then((distance) => {
   scrollDistance = distance;
```

Comment:
Once the website loads, we need to know if the tracking is paused and update the live counter accordingly

Reply:
I think you can have a function in stats.js that the script can call to know if it's paused passing in the domain name: `(domain) => boolean`